meta/.galaxy_install_info

```diff
@@ -0,0 +1 @@
+{install_date: 'Tue Jul  4 10:56:19 2017', version: 2.0.0}
```

Comment:
Hi Jeff, thank you so much for reviewing.
It's my bad that incidentally adding this file. I'll remove it in other commit.
Maybe i could add meta/.galaxy_install_info into .gitignore.